src/frame/sdk/GroupApi.ts

```diff
@@ -114,27 +105,43 @@ function getCompleteListOfUserPublicKeys(userList: string[]) {
     });
 }
 
+/**
+ * Retrieves the master public keys for users in the memberList, adminList and owner.
+ */
+function validateUsers(memberList: string[], adminList: string[], maybeOwnerUserId?: string) {
+    const ownerUserId = maybeOwnerUserId ? [maybeOwnerUserId] : [];
+    const uniqueUsers = Array.from(new Set([...memberList, ...adminList, ...ownerUserId]).values());
```

Comment:
Let's add a comment above this line that says why we're creating a new Set.

Reply:
done